ext/gettext/gettext.c

```diff
@@ -78,12 +78,13 @@ PHP_MINFO_FUNCTION(php_gettext)
 /* {{{ Set the textdomain to "domain". Returns the current domain */
 PHP_FUNCTION(textdomain)
 {
-	char *domain_name = NULL, *retval;
+	char *retval = NULL, *domain_name = NULL;
```

Comment:
what's the purpose of this order change?

Reply:
accidental :)